src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -5,19 +5,25 @@
 import net.minecraft.item.Item;
 import net.minecraftforge.fml.common.registry.IForgeRegistry;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Class to create the modBlocks.
  * References to the blocks can be made here
  * <p>
  * We disabled the following finals since we are neither able to mark the items as final, nor do we want to provide public accessors.
  */
 @SuppressWarnings({"squid:ClassVariableVisibilityCheck", "squid:S2444", "squid:S1444", "squid:S1820",})
+
 public final class ModBlocks
 {
     /*
      * Creating objects for all blocks in the mod.
      * References can be made to here.
      */
+
+    public static List<BlockTimberFrame> timberFrames = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)